src/views/HomeView.vue

```diff
@@ -130,53 +139,42 @@ axios.post('/submit', bet)
 
 <!-- TODO: Figure out how to make this less verbose -->
     <main class="content">
-      <div v-for="sport in eventRows"
-      :key="sport.initial" 
-      class="sportRow">
-        <h2>{{ sport.name }}</h2>
+      <div
+        v-for="{name, initial, allEvents} in eventRows"
+        :key="{initial}"
+        class="sportRow">
+        <h2 class="sportName">{{ name }}</h2>
```

Comment:
In this file, add spaces between equally indented tags. So leave a space between `<template>` tags, etc.